src/components/Spinner/Spinner.stories.tsx

```diff
@@ -1,13 +1,22 @@
 import { Spinner } from './Spinner';
+import { Button } from '../Button/Button';
 
 export default {
   title: 'Components/Spinner',
 };
 
-export const _Spinner = {
+export const _Default = {
   render: () => (
     <>
       <Spinner />
     </>
   ),
 };
+
+export const _Button = {
+  render: () => (
+    <Button>
+      <Spinner variant="button" />
```

Comment:
Question because may I miss the context: instead of having a variant for the button, should be better take a customClass in the spinner only? 🤔 In the end we would have: 

`<Spinner customClass=" via-neutral to-neutral">` and that's all! No!?

Isn't simpler in this case!?